elide-core/src/test/java/com/yahoo/elide/core/LifeCycleTest.java

```diff
@@ -397,6 +412,80 @@ public void testUpdateWithChangeSpec() {
         verify(onUpdateImmediateCallback, times(1)).execute(eq(book), isA(RequestScope.class), any());
     }
 
+    @Test
```

Comment:
Line 412 - we should verify onUpdatePostCOmmitCallback gets called.

Reply:
Done